po/ta.po

```diff
@@ -0,0 +1,7195 @@
+# தமிழ் translation of geany.
+# Copyright (C) 2006 The Geany contributors.
+# This file is distributed under the same license as the geany package.
+# Jeff Bailes <thepizzaking@gmail.com>, 2007-2011.
+#
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Geany 2.0\n"
+"Report-Msgid-Bugs-To: https://github.com/geany/geany/issues\n"
+"POT-Creation-Date: 2023-10-04 23:48+0200\n"
+"PO-Revision-Date: 2014-05-05 05:31+0200\n"
+"Last-Translator: தமிழ்நேரம்<https://TamilNeram.github.io>\n"
+"Language-Team: தமிழ்நேரம்<https://TamilNeram.github.io>\n"
+"Language: ta_IN\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#: geany.desktop.in:5 data/geany.glade:6456
+msgid "Geany"
+msgstr "சீனி"
+
+#: geany.desktop.in:6
+msgid "Integrated Development Environment"
+msgstr "ஒருங்கிணைந்த மேம்பாட்டு சூழல்"
+
+#: geany.desktop.in:7
+msgid "A fast and lightweight IDE using GTK+"
+msgstr "GTK+ ஐப் பயன்படுத்தி வேகமான மற்றும் இலகுரக IDE"
+
+#: geany.desktop.in:14
+msgid "Text;Editor;"
+msgstr "Editor"
+
+#: data/geany.glade:130
+msgid "_Toolbar Preferences"
+msgstr "_Toolbar விருப்பத்தேர்வுகள்"
```

Comment:
Just to be sure there's no misunderstanding: you are allowed to translate the `_Toolbar` part, what the `_` does is mark the next character as being an accelerator, e.g. a letter that if it's key is pressed together with <kbd>Alt</kbd> (or no modifier, in case of menus) will activate the associated UI control.  For example, in this case if the user right-clicks on the toolbar, they'll get a context menu with an item with this text, and can hit <kbd>t</kbd> and it will act as if they clicked the item.

In English and other Latin languages, it's usual to just pick a letter from the words as accelerator, but in languages where letters usually don't have a direct match on the keyboard (e.g. require more complex input methods) like e.g. Chinese, it's usual for translators to explicitly add an easy to type accelerator in parentheses, [like is done in Geany's `zh_CN` translation for this very string](https://github.com/geany/geany/blob/512e88df8d97284c191459450bce4dd7877b9c2b/po/zh_CN.po#L42).  I don't know how Tamil works in that regard, but I guess you would :slightly_smiling_face: